ReactAndroid/src/main/java/com/facebook/react/modules/fresco/FrescoModule.java

```diff
@@ -154,7 +154,7 @@ private static ImagePipelineConfig getDefaultConfig(ReactContext context) {
     HashSet<RequestListener> requestListeners = new HashSet<>();
     requestListeners.add(new SystraceRequestListener());
 
-    OkHttpClient client = OkHttpClientProvider.createClient();
+    OkHttpClient client = new DefaultOkHttpProvider().get();
```

Comment:
If fetching images should not be affected by the HTTP provider from `MainPackageConfig` we should document the `MainPackageConfig` only affects the `fetch` API.

Maybe someone will come up with a request to also use the configured HTTP client to include custom headers, logging etc. when fetching images. Would it make sense for this PR to either:
- Use the same HTTP client in `MainPackageConfig` both for `fetch` and images
- Allow the HTTP client for `fetch` and for images to be configured separately?